### PR TITLE
remove =false from options that default to false in the docs

### PR DESCRIPTION
**Pull request by duglin**
This re-aligns the docs with what the cmd line now does.

Signed-off-by: Doug Davis dug@us.ibm.com

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM :+1: 

ping @thaJeztah @moxiegirl

**Comment by thaJeztah** [comment]
Oh, thanks @duglin :heart_eyes: 

LGTM

hm, guess we need to check the man pages as well

**Comment by duglin** [comment]
darn - I forgot that those weren't under "docs" - I'll add them to this PR - hold on...

**Comment by duglin** [comment]
hmm odd, the man pages are ok as is.

**Comment by duglin** [comment]
never mind - they do need tweaking...  :-)

**Comment by thaJeztah** [comment]
oh, cool!

**Comment by thaJeztah** [comment]
shucks our comments crossed.
want to do that in this PR, or a separate one?

**Comment by duglin** [comment]
might as well just do it here - should be quick

**Comment by vdemeester** [comment]
Back to code review then :stuck_out_tongue_closed_eyes: :santa:

**Comment by duglin** [comment]
ok think I got 'em all

File: man/docker-execwait.1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+% DOCKER(1) Docker User Manuals
```
Comment (thaJeztah) [inline man/docker-execwait.1.md]: ehm, probably added this file by mistake?

**Comment by thaJeztah** [comment]
awesome, @duglin, LGTM

**Comment by vdemeester** [comment]
Still LGTM :wink:

**Comment by thaJeztah** [comment]
let's merge!

**Comment by moxiegirl** [comment]
Gentlemen, I salute you (or Gatsby does). 

http://i.giphy.com/jNdw5Qmy5MOpq.gif

**Pull request merged**